views/benefits/student_loan-en.njk

```diff
@@ -0,0 +1,28 @@
+<div class="m-5 p-5 shadow-result">
+
+  <h2 class="text-2xl font-bold mb-6" id="student_loan">
+    {{__("student_loan.header")}}
```

Comment:
should we not also bring the headers in to here?

Reply:
This string is also used for generating the links on the result page, I figured since that link is identical to the header it made sense to keep this read from the localization files.

Reply:
I'll try this and see if it works.

Reply:
ohhhh no no worries, that makes sense (I tried searching for it else where, but must have missed it)